tools/eos-label-location.in

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
+#
+# eos-label-location: edit the location label for metrics instrumentation daemon
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
```

Comment:
s/(C)/©/, for which you will also need to add the following as the second line of the file:
```
# -*- coding: utf-8 -*-
```